### DISCUSSION TITLE
第6节课笔记(8班)

### DISCUSSION THREAD

**Discussion by Hongru0306** [InternLM/Tutorial#414] [笔记]
请在下方提交笔记链接,格式为
-->报名序号+连接:
例如:213+https://kvudif1helh.feishu.cn/docx/...

**Comment by BJUT-WangZhengTao** [comment]
371+https://blog.csdn.net/m0_54169072/article/details/135712271

**Comment by superkong001** [comment]
545+https://github.com/superkong001/InternLM_Learning/blob/main/homework5_opencompass_note.md

**Comment by Yujin320** [comment]
587+https://github.com/Yujin320/InternLM-hyx-Learning/blob/main/handWrite06.md

**Comment by ZPfree** [comment]
605+https://github.com/ZPfree/homework/blob/main/6note.md

**Comment by baijiesong** [comment]
509+https://github.com/baijiesong/InternLM_Learning/blob/main/%E7%AC%AC%E5%85%AD%E8%8A%82%E8%AF%BE%E7%AC%94%E8%AE%B0.md

**Comment by NYYY0421** [comment]
501+https://github.com/NYYY0421/homework6/blob/main/note6.md

**Comment by hzsun1995** [comment]
446+https://github.com/hzsun1995/internlm-course/blob/main/course6_note.md

**Comment by sunxiaobei** [comment]
585+https://github.com/sunxiaobei/InternLM_tutorial/blob/main/notes/Day06.md

**Comment by Cyydz** [comment]
527+https://www.wolai.com/3oPwED5Ky7L7a9Cfa2vCA6

**Comment by Alias-z** [comment]
388+http://t.csdnimg.cn/8JtvP

**Comment by JieGenius** [comment]
555+https://github.com/JieGenius/InternLMLearning/blob/main/lesson6/note.md

**Comment by zhaoxin111** [comment]
445+https://app.yinxiang.com/fx/87095533-4161-49bf-a257-cee99a871865

**Comment by jamiechoi1995** [comment]
554+https://blog.csdn.net/sinat_26253653/article/details/135773987

**Comment by Doublefei-Giser** [comment]
409+https://blog.51cto.com/u_16507376/9386314

**Comment by carlinl** [comment]
596 + https://pt35a5ibej.feishu.cn/docx/RdLDd1AWqowjozxFxv9crOknnIg?from=from_copylink
